.github/workflows/bats.yaml

```
@@ -209,6 +213,7 @@ jobs:
         RD_USE_GHCR_IMAGES:      "true"
         RD_USE_RAMDISK:          "true"
         RD_USE_WINDOWS_EXE:      "${{ runner.os == 'Windows' }}"
+        RD_USE_NETWORKING_TUNNEL: ${{ runner.os == 'Windows' && inputs.rd-use-networking-tunnel == true }}
```

Does this need quotes to force the value to be a string and not a boolean?

```suggestion
        RD_USE_NETWORKING_TUNNEL: "${{ runner.os == 'Windows' && inputs.rd-use-networking-tunnel == true }}"
```

---

It seemed to work without. But I can force it, sure.